charforge-gui/backend/app/api/auth.py

```
@@ -0,0 +1,85 @@
+    """Register a new user."""
+    try:
+        user = create_user(
+            db=db,
+            username=user_data.username,
+            email=user_data.email,
+            password=user_data.password
+        )
+        return user
+    except HTTPException:
+        raise
+    except Exception as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)